### ISSUE TITLE
Handler panic on connection.

### ISSUE THREAD

**Issue by pirogoeth** [dcu/onetouch-ssh#11]
When the connection is initiated and the program tries to send the approval request, it panics.

```
Sending approval request to your device... panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x0 pc=0x5080e8]

goroutine 1 [running]:
panic(0x8c3260, 0xc820010100)
        /home/sean/.gimme/versions/go1.6.1.linux.amd64/src/runtime/panic.go:464 +0x3e6
github.com/dcu/go-authy.NewApprovalRequest(0xc8202f6870, 0x7f71e979d1a8, 0x0, 0x0)
        /home/sean/.go/src/github.com/dcu/go-authy/approval_request.go:53 +0x1d8
github.com/dcu/go-authy.(*Authy).SendApprovalRequest(0xc8200dbec0, 0x7ffcb1cb0b8a, 0x7, 0xc8203ee180, 0x17, 0xc8203bc1e0, 0xc8203bc210, 0x0, 0x0, 0x0)
        /home/sean/.go/src/github.com/dcu/go-authy/api.go:142 +0x27b
github.com/dcu/onetouch-ssh/ssh.(*Verification).SendOneTouchRequest(0xc8200dbee0, 0x2b, 0x0, 0x0)
        /home/sean/.go/src/github.com/dcu/onetouch-ssh/ssh/verification.go:117 +0x731
github.com/dcu/onetouch-ssh/ssh.(*Verification).Run(0xc8200dbee0, 0x0, 0x0)
        /home/sean/.go/src/github.com/dcu/onetouch-ssh/ssh/verification.go:45 +0x6c
github.com/dcu/onetouch-ssh/cmd.glob.func6(0xbd5320, 0xc8201021f0, 0x1, 0x1)
        /home/sean/.go/src/github.com/dcu/onetouch-ssh/cmd/shell.go:47 +0x8d
github.com/spf13/cobra.(*Command).execute(0xbd5320, 0xc8201021a0, 0x1, 0x1, 0x0, 0x0)
        /home/sean/.go/src/github.com/spf13/cobra/command.go:565 +0x85a
github.com/spf13/cobra.(*Command).ExecuteC(0xbd5120, 0xbd5320, 0x0, 0x0)
        /home/sean/.go/src/github.com/spf13/cobra/command.go:651 +0x55c
github.com/spf13/cobra.(*Command).Execute(0xbd5120, 0x0, 0x0)
        /home/sean/.go/src/github.com/spf13/cobra/command.go:610 +0x2d
github.com/dcu/onetouch-ssh/cmd.Execute()
        /home/sean/.go/src/github.com/dcu/onetouch-ssh/cmd/root.go:42 +0x27
main.main()
        /home/sean/.go/src/github.com/dcu/onetouch-ssh/main.go:26 +0x14
```

Earlier today I was connecting from Putty and the connection would drop immediately without showing any logs.

I'm just filing this bug for historical purposes as I'm going to try to fix it right now.

Ninja edit: Going to attempt to determine if this bug is introduced here or in go-authy.

**Comment by pirogoeth** [comment]
Alright, nailed this down. It happens when an invalid response is received from the Authy API by the onetouch-ssh app, which, in this case is a "there are no devices enabled for onetouch".

This needs to be handled gracefully and maybe dumped to the screen so the user is aware that they need to fix something.

**Comment by pirogoeth** [comment]
Closed this up in #10.
